tasks/leave_restart_consul.yml

```
@@ -0,0 +1,19 @@
+---
+
+- name: Consul leave
+  delegate_to: "{{ consul_servers }}"
```

```suggestion
  delegate_to: "{{ consul_server }}"
```

....if you agree with my corresponding suggestion in `handlers/restart_consul.yml`